supported/osg-htc/osg-hosted-ce/templates/configmap.yaml

```diff
@@ -136,15 +137,15 @@ data:
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: osg-hosted-ce-{{ .Values.Instance }}-slate-scitokens
+  name: osg-hosted-ce-{{ .Values.Instance }}-scitokens
   labels:
     app: osg-hosted-ce
     instance: {{ .Values.Instance }}
     release: {{ .Release.Name }}
     app.kubernetes.io/part-of: {{ .Chart.Name }}
     app.kubernetes.io/instance: {{ .Release.Name }}
 data:
-  50-slate-scitokens.conf: |+
+  50-scitokens.conf: |+
```

Comment:
It's worth looking in the container to make sure there isn't anything hardcoded referencing this file

Reply:
So I searched here: https://github.com/search?q=repo%3Ahtcondor%2Fhtcondor-ce%20scitokens.conf&type=code
And here: https://github.com/search?q=repo%3Aopensciencegrid%2Fdocker-compute-entrypoint%20scitokens.conf&type=code

And don't see any reference at all to this. Is there somewhere else I'm supposed to check? Also what's the difference between that conf and this one: https://github.com/htcondor/htcondor-ce/blob/056e76af28d258db82dc895c0b6a82c02e765fa6/config/mapfiles.d/10-scitokens.conf

Reply:
Looks like we're good as the CE container images look at the entire mapfiles.d dir for mappings to determine which users need to be provisioned: https://github.com/opensciencegrid/docker-compute-entrypoint/blob/master/base/etc/osg/image-config.d/ce-common-startup

The default conf that you link is mostly intended for admins setting up their own local CE and notice that there aren't any uncommented lines. The entire `mapfiles.d` dir is parsed in alphanumeric order and concatenated together to get credential -> user mappings (the file that determines what's included is `/etc/condor-ce/mapfiles.d`). 

You'll see this pattern used a lot for configuration, e.g. `conf.d` dirs, across different Linux applications.